This fixes the wxplot2d plotting issue found at
https://github.com/andrejv/wxmaxima/issues/973.

From 5a0693c97ceaa4935b908f1e478126896952f399 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gunter=20K=C3=B6nigsmann?= <gunter@peterpall.de>
Date: Mon, 19 Feb 2018 05:37:35 +0100
Subject: [PATCH] Seems I accidentally made wxMaxima to default to parameters
 for old gnuplots. Resolves #973

---
 data/wxmathml.lisp.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/wxmathml.lisp.in b/data/wxmathml.lisp.in
index a32e3fc3..4e19acaf 100644
--- a/data/wxmathml.lisp.in
+++ b/data/wxmathml.lisp.in
@@ -43,7 +43,7 @@
 (defvar $wxsubscripts t
   "Recognize TeX-style subscripts")
 (defvar $wxplot_pngcairo nil "Use gnuplot's pngcairo terminal for new plots?")
-(defmvar $wxplot_old_gnuplot t)
+(defmvar $wxplot_old_gnuplot nil)
 
 (defun $wxstatusbar (status)
   (format t "<statusbar>~a</statusbar>~%" status))
